joplin/pages/service_page/fixtures/helpers/components.py

```diff
@@ -6,11 +6,20 @@
 import pages.location_page.fixtures as location_page_fixtures
 import pages.topic_page.fixtures as topic_page_fixtures
 import pages.service_page.fixtures as service_page_fixtures
+from pages.home_page.factories import HomePageFactory
 
 
 def home():
-    return HomePage.objects.first()
```

Comment:
Was a HomePage not returning? There should've been one created in migrations.

Reply:
it was not. it was returning as None in my isolated user tests, which is why I check to see if exists and if not, making a new one

Reply:
also, this runs with tests, do migrations run before we run pytest?